docs/design.md

```diff
@@ -104,6 +104,15 @@ These blocks are called _address blocks_, and assigned to nodes.  Since all IP a
 
 For example, if an address pool defines that the size of an address block is 2<sup>5</sup>, `coil-controller` will curve an address block for IPv4 with `/27` subnet mask out of the pool, and assigns it to a node.
 
+When assigning an address, if `coil-controller` assigns the last released address again, it is possible to use old routes and it is not possible to establish a new connection via egress NAT.
+Therefore, avoiding reassigning the last released address, it records the index of the last assigned address.
+And in the next time, it assigns an address next to the recorded index.
+
+When we use an address block with the size `/32`, it may occur that reassignment of the same address block.
+However, the address block with the size `/32` is usually used as a global address.
+A global address will not frequently allocate and release.
+Thus, we don't implement this.
```

Comment:
But if it happens, does Cilium work well?

Reply:
In this situation, I think Cilium cannot handle the connection well.
But the frequency may be less than the case of reusing addresses.

As a result of the discussion, if we just address the issue we face now, considering the cost and time of fix, we don't need to deal with address blocks case.

Should we deal with in this fix?

Reply:
I'm not against you and the team, but I'm a bit concerned that the loss of connectivity would be more severe for global addresses rather than cluster addresses.

Reply:
Now I think it is enough that we are aware of this problem and we mention it in this document.
In the long term, I also think that it is better to fix this.

Reply:
ok